src/backend/marsha/core/serializers/file.py

```diff
@@ -173,3 +175,32 @@ class InitiateUploadSerializer(serializers.Serializer):
 
     filename = serializers.CharField()
     mimetype = serializers.CharField(allow_blank=True)
+
+
+class SharedLiveMediaInitiateUploadSerializer(InitiateUploadSerializer):
+    """An initiate-upload serializer dedicated to shared live media."""
+
+    def validate(self, attrs):
```

Comment:
Is it possible to test this code, I don't see any test

Reply:
This is all tested in `test_api_shared_live_media.py` file. All tests about `initiate-upload` endpoint.

Reply:
And I don't really understand what you mean by `the two cases of mime type` ?